api/src/main/java/org/openmrs/module/fhir2/api/translators/LocationTranslator.java

```diff
@@ -24,6 +27,15 @@ public interface LocationTranslator extends OpenmrsFhirUpdatableTranslator<org.o
 	@Override
 	Location toFhirResource(@Nonnull org.openmrs.Location openmrsLocation);
 	
+	/**
+	 * Maps a collection of {@link org.openmrs.Location}s to a {@link org.hl7.fhir.r4.model.Location}
+	 *
+	 * @param openmrsLocations the collection of locations to translate
+	 * @return the mapping of OpenMRS location to corresponding FHIR location resource
+	 */
+	@Override
+	Map<org.openmrs.Location, Location> toFhirResources(Collection<org.openmrs.Location> openmrsLocations);
```

Comment:
I'm not sure about this return type. Is there some reason this can't just return a `List<Location>` or `Collection<Location>` or similar? While there's clear value to a map inside the implementation, I have a harder time grasping the use-case for this map.

Reply:
I've started with a List - modified the `org.openmrs.module.fhir2.api.translators.ToFhirTranslator` interface.

But it wouldn't work with `org.openmrs.module.fhir2.api.translators.LocationTypeTranslator#toFhirResource` that returns a `List<CodeableConcept>`, the `CodeableConcept` have to back-reference to Location, so the caller of `toFhirResources(Collection<Location>)` had no way to group them.

I was considering wrapping `List<CodeableConcept>` into some object with Location/Location.id, but I went with "don't overthink, just make it work" solution.